docs/quickstart.html

```diff
@@ -32,7 +32,7 @@ <h4 class="anchor-heading">
             the latest Kafka release and extract it:
         </p>
 
-        <pre class="line-numbers"><code class="language-bash">$ tar -xzf kafka_{{scalaVersion}}-{{fullDotVersion}}.tgz
+        <pre class="language-bash"><code class="language-bash">$ tar -xzf kafka_{{scalaVersion}}-{{fullDotVersion}}.tgz
```

Comment:
The `<code>` already has `class="language-bash"`, so I don't think we need it on the `<pre>` as well.

Reply:
<img width="684" alt="image" src="https://github.com/apache/kafka/assets/13068144/e4285029-5a1c-4002-8ed6-f8a95c97e2a0">

If I remove the `language-bash` class from `<pre>` it will lose the associated properties as shown in the first image above